lib/benchpark/test_repo/repo.yaml

```diff
@@ -0,0 +1,2 @@
+repo:
```

Comment:
We need to move this repo to a reasonable place now that it works

Reply:
(I think this is resolved along with https://github.com/LLNL/benchpark/pull/298#discussion_r1714436173, but IMO that comment tracks this same issue if it is not resolved)